src/flamenco/runtime/tests/fd_exec_sol_compat.c

```diff
@@ -163,8 +177,7 @@ sol_compat_execute_wrapper( fd_exec_instr_test_runner_t * runner,
       *output = NULL;
       break;
     }
-  } while(0);
-  // fd_scratch_pop();
+  } FD_SCRATCH_SCOPE_END;
```

Comment:
IIRC these functions publish in to the caller's scratch frame so freeing the frame here could cause memory corruption (use after free)

Reply:
@ripatel-fd https://github.com/firedancer-io/firedancer/compare/7421edee062eace051669dd85a7f8d20110d3bc7..c810ce395d4b828970bce8c441e6a3ddcaeec0b5

I think this just fixes the obvious use-after-free. Will probably need to do a bit of restructuring with how we use scratch frames here at some point.